docs/es/docs/tutorial/query-params.md

```diff
@@ -3,7 +3,7 @@
 Cuando declaras otros parámetros de la función que no hacen parte de los parámetros de path estos se interpretan automáticamente como parámetros de "query".
 
 ```Python hl_lines="9"
```

Comment:
@luigiPinna per the Issue #12554 I think you'd also need to remove ` ```Python hl_lines="9"`

Reply:
Thank you @lenmazzone, now removed